src/product.jl

```diff
@@ -210,7 +210,7 @@ function FusionStyle(::Type{<:ProductSector{T}}) where {T <: SectorTuple}
     return Base.:&(map(FusionStyle, _sectors(T))...)
```

Comment:
Can you also change this one to
```julia
return mapreduce(FusionStyle, &, _sectors(T))
```
(Somehow I am unable to make actual code suggestions.